internal/test_cases/highlighting_test_case.go

```diff
@@ -43,15 +43,19 @@ func (c HighlightingTestCaseCollection) Run(stageHarness *test_case_harness.Test
 
 		// Add pattern
 		allArguments = append(allArguments, []string{"-E", testCase.Pattern}...)
+		pipeToCat := ""
 
 		if testCase.RunInsideTty {
 			logger.Infof("Running grep inside TTY")
```

Comment:
@UdeshyaDhungana let's remove this log entirely, not very useful (from a user's perspective, if we aren't doing `| cat` it should be within a PTY)